src/plugin.js

```diff
@@ -102,54 +85,6 @@ class Cmcd extends Plugin {
 
 }
 
-function generateUuid() {
```

Comment:
CML exports a `uuid` function specifically for use in generating session and content IDs.